core/client/controllers/posts/post.js

```diff
@@ -2,6 +2,20 @@ var PostController = Ember.Controller.extend({
     isPublished: Ember.computed.equal('model.status', 'published'),
     classNameBindings: ['model.featured'],
 
+    authorName: Ember.computed('model.author', function () {
+        return this.get('model.author.name') ? this.get('model.author.name') : this.get('model.author.email');
+    }),
+
+    authorAvatar: Ember.computed('ghostPaths', 'model.author.image', function () {
+        var avatarSrc = null;
+        if (this.get('model.author.image')) {
+            avatarSrc = this.get('model.author.image');
+        } else {
+            avatarSrc = this.get('ghostPaths.url').asset('/shared/img/user-image.png');
+        }
+        return avatarSrc;
+    }),
```

Comment:
```
authorAvatar: Ember.computed('model.author.image', function () {
    return this.get('model.author.image') || this.get('ghostPaths.url').asset('/shared/img/user-image.png');
}),
```

`ghostPaths` won't change so we don't need to depend on it.  The body can be simplified as well but that's just personal preference, so if you prefer the way you have it, don't change it on my account.